astrosat_users/serializers/serializers_users.py

```diff
@@ -13,8 +13,8 @@ class UserSerializerLite(serializers.ModelSerializer):
 
     class Meta:
         model = User
-        fields = ("email", "name", "username", "change_password")
-        read_only_fields = ("email", "name", "username", "change_password")
+        fields = ("email", "name", "username", "change_password", "is_verified", "is_approved", "accepted_terms")
+        read_only_fields = ("email", "name", "username", "change_password", "is_verified", "is_approved", "accepted_terms")
```

Comment:
Do you need to have both `fields` and `read_only_fields`?

Reply:
`read_only_fields` protects me from ever using this serializer in a read-write context.

Reply:
That makes sense but what I was asking was if a field is listed as a `read_only_field` does it _also_ need to be in `fields`?

Reply:
It's the other way around; every field that is part of the serializer needs to be in `fields`; those fields that are read-only should also be in `read_only_fields`.